pkg/bundle/bundle.go

```diff
@@ -31,12 +31,7 @@ const shpIgnoreFilename = ".shpignore"
 // remote.Option for optional options to the image push to the registry, for
 // example to provide the appropriate access credentials.
 func PackAndPush(ref name.Reference, directory string, options ...remote.Option) (name.Digest, error) {
-	r, err := Pack(directory)
-	if err != nil {
-		return name.Digest{}, err
-	}
-
-	bundleLayer, err := tarball.LayerFromReader(r)
+	bundleLayer, err := tarball.LayerFromFile(directory)
```

Comment:
Please note, the `Pack` function has special semantics:
```go
// Pack reads a directory and creates a tar stream with its content by:
// - storing all directories and regular files as-is,
// - dereferencing all symlinks and storing the respective target,
// - ignoring all files configured in .shpignore
func Pack(directory string) (io.Reader, error)
```

Unless `LayerFromFile` supports a way to define a custom ignore file, we have to stick with `Pack` for the time being.

Reply:
I reverted the change by specifying //nolint, just to ignore the linting error.